src/middlewared/middlewared/rclone/remote/s3.py

```diff
@@ -24,7 +24,10 @@ class S3RcloneRemote(BaseRcloneRemote):
     ]
 
     task_schema = [
-        Str("encryption", enum=[None, "AES256"], default=None, null=True),
+        Str("encryption", title="Server-Side Encryption", enum=[None, "AES256"], default=None, null=True),
+        Str("storage_class", title="The storage class to use", enum=["", "STANDARD", "REDUCED_REDUNDANCY",
+                                                                     "STANDARD_IA", "ONEZONE_IA", "GLACIER",
+                                                                     "DEEP_ARCHIVE"]),
```

Comment:
How does this work with non-AWS? Shoudnt we have a default here?

Reply:
It's not required, so we don't need a default. `""` is an rclone default and it works with non-AWS.